homeassistant/components/efergy/sensor.py

```diff
@@ -45,12 +47,14 @@
         name="Energy Usage",
         device_class=DEVICE_CLASS_POWER,
         native_unit_of_measurement=POWER_WATT,
+        state_class=STATE_CLASS_MEASUREMENT,
     ),
     CONF_AMOUNT: SensorEntityDescription(
         key=CONF_AMOUNT,
         name="Energy Consumed",
         device_class=DEVICE_CLASS_ENERGY,
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
```

Comment:
It would be good to update the tests to assert the state attributes and check device class, unit and state class of the different measurements.